SimG4CMS/Calo/src/ECalSD.cc

```diff
@@ -225,14 +226,13 @@ int ECalSD::getTrackID(G4Track* aTrack) {
 
 uint16_t ECalSD::getDepth(G4Step * aStep) {
   G4LogicalVolume* lv   = aStep->GetPreStepPoint()->GetTouchable()->GetVolume(0)->GetLogicalVolume();
-  uint16_t ret = 0;
-  if (any(useDepth1,lv))      ret = 1;
-  else if (any(useDepth2,lv)) ret = 2;
-  else if (storeRL) ret = getRadiationLength(aStep);
-#ifdef DebugLog
-  LogDebug("EcalSim") << "Volume " << lv->GetName() << " Depth " << ret;
-#endif
-  return ret;
+  if      (storeRL) {
```

Comment:
hi @bsunanda @civanch - the reordering of this if statement is naively a behavior change. What is the purpose?